examples/forecasting/_tirex_test.py

```diff
@@ -0,0 +1,9 @@
+from sktime.datasets import load_airline
```

Comment:
this is tested by the automated framework - you do not need to add this file (was this committed accidentally?)